### PR TITLE
Replace Mazer references with ansible-galaxy

### PR DESCRIPTION
**Pull request by jborean93**
##### SUMMARY
Replace the Mazer references in the collections tech preview with more info around `ansible-galaxy collection ...`.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ansible-galaxy

### CODE REVIEW

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 1]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
-.. Note::
-    All the documentation below that use ``mazer`` might be updated to use another tool in the future as ``mazer`` will not be updated in the future.
+.. note::
+    Any references to ``ansible-galaxy`` below will be of a 'working version' that is in development for the 2.9
+    release. As such, the command and this documentation section is subject to frequent changes.
+
+Currently the ``ansible-galaxy collection`` command implements the following sub commands:
 
-We are working on integrating this into Ansible itself for 2.9. Currently we have an `ansible-galaxy PR <https://github.com/ansible/ansible/pull/57106>`_ incorporating some of the commands into ``ansible-galaxy``. Currently it is not installable outside Ansible, but we hope to land this into development soon so early adopters can test.
+* ``init``: Create a basic collection skeleton based on the default template included with Ansible or your own
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``init``: Create a basic collection skeleton based on the default template included with Ansible or your own.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 2]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
+* ``build``: Create a collection artifact that can be uploaded to Galaxy or your own repository
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``build``: Create a collection artifact that can be uploaded to Galaxy or your own repository.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 3]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
+* ``publish``: Publish a built collection artifact to Galaxy
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``publish``: Publish a built collection artifact to Galaxy.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 4]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
+* ``install``: Install one or multiple collections
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``install``: Install one or multiple collections.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 5]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
 
-.. Note::
-    Any references to ``ansible-galaxy`` below will be of a 'working version' either in this PR or subsequently in development. As such, the command and this documentation section is subject to frequent change.
+You can learn more about the ``ansible-galaxy`` cli tool by read the :ref:`ansible-galaxy` man page._
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
To learn more about the ``ansible-galaxy`` cli tool, read the :ref:`ansible-galaxy` man page.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 6]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
 
 In the end, to get started with authoring a new collection it should be as simple as:
 
 .. code-block:: bash
 
-    collection_dir#>ansible-galaxy collection init
+    collection_dir#> ansible-galaxy collection init namespace.collection
 
+And then populating the directories with the content you want inside the collection. You can also have a look at
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
And then populating the directories with the content you want inside the collection. See
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 7]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
 
 .. _building_collections:
 
 Building collections
 ====================
 
-Collections are built by running ``mazer build`` from inside the collection's root directory.
-This will create a ``releases/`` directory inside the collection with the build artifacts,
-which can be uploaded to Galaxy.::
+Collections are built by running ``ansible-galaxy collection build`` from inside the collection's root directory. This
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
Run ``ansible-galaxy collection build`` from inside the collection's root directory to build a collection. This
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 8]
```diff
@@ -113,54 +113,74 @@ TBD. Expect tests for the collection itself to reside here.
...
+will create a tarball of the built collection in the current directory which can be uploaded to Galaxy.::
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
creates a tarball of the built collection in the current directory which can be uploaded to Galaxy.::
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 9]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
 
-You can also keep a collection adjacent to the current playbook, under a ``collections/ansible_collection/`` directory structure.
+You can also keep a collection adjacent to the current playbook, under a ``collections/ansible_collections/`` directory structure.
 
 ::
 
     play.yml
     ├── collections/
-    │   └── ansbile_collection/
+    │   └── ansbile_collections/
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
    │   └── ansible_collections/
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 10]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
     │               └── myname/
     │                   └── mycol/<collection structure lives here>
 
 
+By default ``ansible-galaxy`` will install the latest collection that is available but a version range identifier can
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
By default ``ansible-galaxy`` installs the latest collection that is available but  you can add a version range identifier 
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 11]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+be used to filter the version that is installed like so:
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
to install a specific version as follows:
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 12]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+   ansible-galaxy collection install mynamespace.mycollection:1.0.0
+
+   # Install the collection at 1.0.0-beta.1
+   ansible-galaxy collection install mynamespace.mycollection:==1.0.0-beta.1
+
+   # Only install the collections that are greater than or equal to 1.0.0 or less than 2.0.0
+   ansible-galaxy collection install mynamespace.mycollection:>=1.0.0,<2.0.0
+
+
+Multiple range identifiers can be specified and are split by ``,``. The following range identifiers can be used:
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
You can specify multiple range identifiers which are split by ``,``. You can use the following range identifiers:
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 13]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+
+* ``*``: Any version, this is the default used when no range specified is set.
+* ``!=``: Version is not equal to the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``!=``: Version is not equal to the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 14]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+* ``==``: Version must be the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``==``: Version must be the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 15]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+* ``>=``: Version is greater than or equal to the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``>=``: Version is greater than or equal to the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 16]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+* ``>``: Version is greater than the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``>``: Version is greater than the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 17]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+* ``<=``: Version is less than or equal to the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``<=``: Version is less than or equal to the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 18]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+* ``<``: Version is less than the one specified
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
* ``<``: Version is less than the one specified.
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 19]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
+
+.. note::
+    The ansible-galaxy command will ignore any pre-release versions unless the ``==`` range identifier is used to
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
    The ``ansible-galaxy`` command  ignores any pre-release versions unless the ``==`` range identifier is used to
```

File: docs/docsite/rst/dev_guide/collections_tech_preview.rst [hunk 20]
```diff
@@ -208,30 +212,86 @@ The recommended way to install a collection is:
...
 
 assuming the collection is hosted in Galaxy.
 
 You can also use a tarball resulting from your build:
 
 .. code-block:: bash
 
-   #> ansible-galaxy install mynamespace.mycollection.0.1.0.tgz -p /path
+   # Will install the collection to ./collections/ansible_collections/mynamespace/mycollection
+   ansible-galaxy install mynamespace-mycollection-0.1.0.tar.gz -p ./collections/ansible_collections
```
Comment (samccann) [inline docs/docsite/rst/dev_guide/collections_tech_preview.rst]:
```suggestion
   ansible-galaxy collection install mynamespace-mycollection-0.1.0.tar.gz -p ./collections/ansible_collections
```

**Review by samccann — COMMENTED** [review]
looks good! Just some docs nit suggestions.

**Comment by jborean93** [comment]
Thanks @samccann for the review, just pushed the changes now.

**Review by samccann — APPROVED** [review]

**Pull request merged**